pkgs/development/python-modules/python-poppler/default.nix

```diff
@@ -0,0 +1,62 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pythonOlder,
+  pytestCheckHook,
+  setuptools,
+  meson-python,
+  ninja,
+  meson,
+  poppler,
+  pkg-config,
+  pybind11,
+}:
+
+buildPythonPackage rec {
+  pname = "python-poppler";
+  version = "0.4.1";
+  pyproject = true;
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "python_poppler";
+    hash = "sha256-5spcI+wCNQvyzvhaa/nxsmF5ZDbbR4F2+dJPsU7uzGo=";
+  };
+
+  patches = [
+    # Prevent Meson from downloading pybind11, use system version instead
+    ./use_system_pybind11.patch
+  ];
+
+  build-system = [ setuptools ];
+
+  buildInputs = [ pybind11 ];
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+  ];
+
+  dependencies = [
+    meson-python
+    poppler
```

Comment:
`poppler` is not a python dependency, so it should go to `propagatedBuildInputs`.